### ISSUE TITLE
Print the revision name after service create and update commands

### ISSUE THREAD

**Issue by navidshaikh** [knative/client#290]
> /kind feature

## Describe the feature:

When a user updates a service, we print the success message
```
➜ kn service update bluegreen --env TARGET=green
Waiting for service 'bluegreen' to become ready ... OK
Service 'bluegreen' updated in namespace 'default'.
```

however, we should also print the revision name generated after this update, which can be used to refer that revision for further operations, eg: traffic splitting. This can also be used for scripting use cases.

Update:
Implement same behavior for `service create` command as well.

**Comment by duglin** [comment]
they should also be allowed to specify the new traffic split on the update cmd

**Comment by navidshaikh** [comment]
> they should also be allowed to specify the new traffic split on the update cmd

sorry I didn't get that, could you please add some more detail  ?

**Comment by duglin** [comment]
Sorry I really should have put that comment on the other issue - see: https://github.com/knative/client/pull/285#issuecomment-514602694

**Comment by rhuss** [comment]
> Implement same behavior for `service create` command as well.

For `kn service create` we print the URL at the end. The service name does not make sense, as its provided as argument.

With having the URL on a single line as last argument you can do things like

```
curl $(kn service create hello --image ... | tail -1)
```

Not sure if this  a good thing when people will rely on that or if  a `kn service describe hello --url` is better suited (I tend to the latter)

**Comment by navidshaikh** [comment]
Before printing the URL, we can add a line may be saying
```
Created revision 'svc-abcde'
[..]
```
keeping the URL line at the bottom only.
WDYT ?

**Comment by duglin** [comment]
I agree with @rhuss on the `tail` thing - I tend to use stuff like that in bash-scripts all the time.

**Comment by navidshaikh** [comment]
@duglin @rhuss : the `tail` use case is intact, I am saying to have a line in output (not the last line) printing revision name.. like
```
✗  kn service create hello --image docker.io/swordphilic/go-helloworld
Service 'hello' successfully created in namespace 'default'.
Revision 'hello-asdf' created..
Waiting for service 'hello' to become ready ... OK

Service URL:
http://hello.default.apps-crc.testing
```

```
✗  kn service update hello --env TARGET=v2                            
Revision hello-wxyz created ...
Waiting for service 'hello' to become ready ... OK
Service 'hello' updated in namespace 'default'.
```

WDYT ?

**Comment by duglin** [comment]
@navidshaikh looks good - easily parse-able by scripts  :-) 

w.r.t. the tail thing... I'm ok with it "as is", but if people are concerned about other things possibly showing up in the output after the URL, then we could make it:
```
Service URL: http://hello.default.apps-crc.testing
```
Probably safer for grep-ing.  Just a thought.

**Comment by rhuss** [comment]
This is now already implemented with #431
